File: mambo-protocol/src/main/scala/org/mambo/protocol/enums/NicknameErrorEnum.scala




















// Generated on 05/02/2013 20:27:17
// Scala version by Blackrush
package org.mambo.protocol.enums

import org.mambo.protocol.Enumeration

object NicknameErrorEnum extends Enumeration {
  val alreadyUsed = Value(1, "alreadyUsed")
  val sameAsLogin = Value(2, "sameAsLogin")
  val tooSimilarToLogin = Value(3, "tooSimilarToLogin")
  val invalidNick = Value(4, "invalidNick")
  val unknownNickError = Value(99, "unknownNickError")
}

File: mambo-protocol/src/main/java/org/mambo/protocol/enums/NicknameErrorEnum.java


// Generated on 05/08/2013 19:04:44
package org.mambo.protocol.enums;

public enum NicknameErrorEnum {
    ALREADY_USED(1),
    SAME_AS_LOGIN(2),
    TOO_SIMILAR_TO_LOGIN(3),
    INVALID_NICK(4),
    UNKNOWN_NICK_ERROR(99);
    
    private int value;
    NicknameErrorEnum(int value) { this.value = value; }
    public int value() { return value; }
    
    public static NicknameErrorEnum valueOf(int value) {
    	for (NicknameErrorEnum a : values()) {
    		if (a.value == value)
    			return a;
    	}
    	return null;
    }
    
}
